src/cloud/digitalocean/provisioner.ts

```diff
@@ -149,18 +153,18 @@ class Provisioner {
       this.sendStatus_("OAUTH_INIT");
       oauth.initiateOAuth(REDIRECT_URIS).then((obj: any) => {
         var url = "https://cloud.digitalocean.com/v1/oauth/authorize?" +
-                  "client_id=c16837b5448cd6cf2582d2c2f767cfb7d11844ec395a91b43f26ca72513416c8&" +
-                  "response_type=token&" +
-                  "redirect_uri=" + encodeURIComponent(obj.redirect) + "&" +
-                  "state=" + encodeURIComponent(obj.state) + "&" +
-                  "scope=read%20write";
+          "client_id=c16837b5448cd6cf2582d2c2f767cfb7d11844ec395a91b43f26ca72513416c8&" +
+          "response_type=token&" +
+          "redirect_uri=" + encodeURIComponent(obj.redirect) + "&" +
+          "state=" + encodeURIComponent(obj.state) + "&" +
+          "scope=read%20write";
         return oauth.launchAuthFlow(url, obj);
       }).then((responseUrl: string) => {
         var query = responseUrl.substr(responseUrl.indexOf('#') + 1),
-            param: string,
-            params: { [k: string]: string } = {},
-            keys = query.split('&'),
-            i = 0;
+        param: string,
```

Comment:
4 spaces indentation (we don't use this capability of `var` anywhere else in uproxy-lib but I think 4 is more appropriate)